src/main/java/com/gildedrose/Conjured.java

```diff
@@ -1,27 +1,36 @@
 package com.gildedrose;
 
 public class Conjured extends DefaultItem {
```

Comment:
Question: I see that `DefaultItem` implements `Ageable`, do we still need to extend `DefaultItem` here?

Reply:
Even though Default implements Ageable, we still need the extend because both Conjured and RedWine use the default items quality coefficient to base theirs off of EG: Conjured degrades twice as fast as Normal items (Default Items)

Reply:
Ah, so we are extending for access to the parent's field values?

Reply:
Follow up question: why do some Items inherit while others do not?  It makes thinking of them as a type hierarchy, or as a group of related types confusing.

Reply:
The items that use inheritance are items whos quality coefficient is inherently coupled with the quality coefficient of the Default (normal) item. So while they all implement the ageable interface (they all use age, are all in a ageable grouping), the only ones that inherit are the items that have functionality based on the Default Item. I hope this all makes sense, and thank you for the review! 😄

Reply:
Suggestion: Can we store the default values in a class or enum and then use them in places that make sense? For example:

```java
class Coefficient {
   public final static int QUALITY = 1;
  ...
}

class Conjured implements Ageable {  
  ...
  this.qualityCoefficient = Coefficients.QUALITY * 2;
}
```

This way we won't have to deal with the confusion around what and why some items extend `DefaultItem`. We can also update the coefficients in one spot, and be updated in items that use it.

Reply:
We like the idea, we will for sure look into this for implementing future features and will keep a note/ticket to potentially implement it moving forward.